packages/core/src/App/Components/Routes/binary-routes.jsx

```diff
@@ -24,7 +22,7 @@ const BinaryRoutes = observer(props => {
         <React.Suspense fallback={<Loading />}>
             <Prompt when={prompt_when} message={promptFn} />
             <Switch>
-                {getRoutesConfig({ is_appstore, is_eu_country }).map((route, idx) => (
+                {getRoutesConfig({ is_appstore: false, is_eu_country }).map((route, idx) => (
```

Comment:
do we need is_appstore ?

Reply:
Thanks for highlighting this. 
I have removed this.